ginivision/src/androidTest/java/net/gini/android/vision/uiautomator/screens/Screen.java

```diff
@@ -0,0 +1,5 @@
+package net.gini.android.vision.uiautomator.screens;
+
+public interface Screen {
```

Comment:
I don't understand the purpose of this interface. Maybe a comment or a more descriptive name would help.

EDIT: OK, now I do, by looking at the classes at implement it. But still, it could be more descriptive.